control-plane/README.md

```diff
@@ -1,2 +1,17 @@
-# Control Plane
+
+# Control plane
+
 The control plane is used for controlling and monitoring the jobs.
+
+## Get Started
```

Comment:
Can you use the Firebase emulator to run the job control plane locally?
Examples of instructions:
https://github.com/GoogleChromeLabs/trust-safety-demo/tree/main/conversion-measurement#set-and-run-locally